packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain.tsx

```diff
@@ -744,22 +744,8 @@ export function TransferPanelMain({
             and reach out in #support for assistance.
           </span>
         )
-
-      case TransferReadinessRichErrorMessage.TOKEN_WITHDRAW_ONLY:
-      case TransferReadinessRichErrorMessage.TOKEN_TRANSFER_DISABLED:
```

Comment:
I think we can still have this as a fail-safe. The reason I say this is to avoid states where a withdraw-only token could be selected in deposit mode, but the transfer button doesn't get activated and the user is left clueless without any messaging.
Steps to reproduce:
1. Be in withdrawal mode
2. Search for a withdraw-only token . eg. ZRX. It will get selected.
3. Now switch to deposit mode using the double-arrow button. 
4. You enter the deposit mode with a withdraw-only token selected 😅  but it doesn't let you deposit because the transfer button is disabled.

<img width="200" alt="image" src="https://github.com/OffchainLabs/arbitrum-token-bridge/assets/7558499/5f638322-e2fb-4028-8ca6-0363fc97dccc">